OpenEdXMobile/src/main/java/org/edx/mobile/module/registration/model/RegistrationFormField.java

```diff
@@ -48,8 +50,13 @@ public String getInstructions() {
         return instructions;
     }
 
+    @NonNull
     public RegistrationFieldType getFieldType() {
-        return fieldType;
+        if (fieldType != null) {
```

Comment:
These lines can be optimized to 
        `return fieldType != null ? fieldType : RegistrationFieldType.UNKNOWN;`